### PR TITLE
fix hugegraph-example/Example checkstyle issue

### PR DESCRIPTION
**Pull request by mayelena**
fix hugegraph-example/Example checkstyle issue

### CODE REVIEW

**Review by seagle-yuan — CHANGES_REQUESTED** [review]

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 1]
```diff
@@ -270,7 +270,7 @@ public static void testQuery(final HugeGraph graph) {
         GraphTraversal<Vertex, Vertex> vertices = graph.traversal().V();
         int size = vertices.toList().size();
         assert size == 12;
-        LOG.info(">>>> query all vertices: size=" + size);
+        LOG.info(">>>> query all vertices: size=", size);
```
Comment (seagle-yuan) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:274]: we need use [LOG.info(">>>> query all vertices: size {}",size);]

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 2]
```diff
@@ -382,39 +382,39 @@ public static void testQuery(final HugeGraph graph) {
...
-        LOG.info(">>>> query all persons age<19: " + vertexList);
+        LOG.info(">>>> query all persons age<19: {}", vertexList);
 
         String addr = "Bay Area";
         vertices = graph.traversal().V().hasLabel("author")
                         .has("lived", Text.contains(addr));
         vertexList = vertices.toList();
         assert vertexList.size() == 1;
-        LOG.info(String.format(">>>> query all authors lived %s: %s",
+        LOG.info(String.format(">>>> query all authors lived {}: {}",
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:417]: need to remove "String.format(" here

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 3]
```diff
@@ -268,19 +270,19 @@ public static void testQuery(final HugeGraph graph) {
...
         assert size == 5;
-        System.out.println(">>>> query all persons: size=" + size);
+        LOG.info(">>>> query all persons: size {}", size);
 
         // query vertex by primary-values
         vertices = graph.traversal().V().hasLabel("author").has("id", 1);
         List<Vertex> vertexList = vertices.toList();
         assert vertexList.size() == 1;
-        System.out.println(">>>> query vertices by primary-values: " +
+        LOG.info(">>>> query vertices by primary-values: {}",
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:285]: please update alignment of the next line

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 4]
```diff
@@ -291,19 +293,19 @@ public static void testQuery(final HugeGraph graph) {
...
-        System.out.println(">>>> query vertices of vertex: " + vertexList);
+        LOG.info(">>>> query vertices of vertex: {}", vertexList);
 
         // query edge by sort-values
         vertices = graph.traversal().V(authorId);
         edgesOfVertex = vertices.outE("write").has("time", "2017-4-28");
         edgeList = edgesOfVertex.toList();
         assert edgeList.size() == 2;
-        System.out.println(">>>> query edges of vertex by sort-values: " +
+        LOG.info(">>>> query edges of vertex by sort-values: " +
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:308]: ditto

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 5]
```diff
@@ -355,10 +357,10 @@ public static void testQuery(final HugeGraph graph) {
 
         Iterator<Edge> edges2 = graph.edges(q);
         assert edges2.hasNext();
-        System.out.println(">>>> queryEdges(id-condition): " +
+        LOG.info(">>>> queryEdges(id-condition): {}",
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:360]: ditto

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 6]
```diff
@@ -355,10 +357,10 @@ public static void testQuery(final HugeGraph graph) {
...
                            edges2.hasNext());
         while (edges2.hasNext()) {
-            System.out.println(">>>> queryEdges(id-condition): " +
+            LOG.info(">>>> queryEdges(id-condition): {}",
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:363]: ditto

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 7]
```diff
@@ -368,10 +370,10 @@ public static void testQuery(final HugeGraph graph) {
             q.key(HugeKeys.PROPERTIES, contribution.id());
             Iterator<Edge> edges3 = graph.edges(q);
             assert edges3.hasNext();
-            System.out.println(">>>> queryEdges(contribution): " +
+            LOG.info(">>>> queryEdges(contribution): {}",
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:373]: ditto

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 8]
```diff
@@ -368,10 +370,10 @@ public static void testQuery(final HugeGraph graph) {
...
                                edges3.hasNext());
             while (edges3.hasNext()) {
-                System.out.println(">>>> queryEdges(contribution): " +
+                LOG.info(">>>> queryEdges(contribution): {}",
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:376]: ditto

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example2.java [hunk 9]
```diff
@@ -58,41 +58,41 @@ public static void traversal(final HugeGraph graph) {
         GraphTraversalSource g = graph.traversal();
 
         GraphTraversal<Vertex, Vertex> vertices = g.V();
-        System.out.println(">>>> query all vertices: size=" +
+        LOG.info(">>>> query all vertices: size=" +
                            vertices.toList().size());
 
         List<Edge> edges = g.E().toList();
-        System.out.println(">>>> query all edges: size=" +
+        LOG.info(">>>> query all edges: size=" +
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example2.java:65]: ditto

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example2.java [hunk 10]
```diff
@@ -58,41 +58,41 @@ public static void traversal(final HugeGraph graph) {
         GraphTraversalSource g = graph.traversal();
 
         GraphTraversal<Vertex, Vertex> vertices = g.V();
-        System.out.println(">>>> query all vertices: size=" +
+        LOG.info(">>>> query all vertices: size=" +
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example2.java:61]: can we also use "... size={}" style?

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example2.java [hunk 11]
```diff
@@ -58,41 +58,41 @@ public static void traversal(final HugeGraph graph) {
...
 
         List<Edge> edges = g.E().toList();
-        System.out.println(">>>> query all edges: size=" +
+        LOG.info(">>>> query all edges: size=" +
                            edges.size());
 
         List<Object> names = g.V().inE("knows").limit(2)
                               .outV().values("name").toList();
-        System.out.println(">>>> query vertex(with props) of edges: " + names);
+        LOG.info(">>>> query vertex(with props) of edges: " + names);
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example2.java:70]: can we also use "{}" style?

File: hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java [hunk 12]
```diff
@@ -282,8 +282,7 @@ public static void testQuery(final HugeGraph graph) {
         vertices = graph.traversal().V().hasLabel("author").has("id", 1);
         List<Vertex> vertexList = vertices.toList();
         assert vertexList.size() == 1;
-        LOG.info(">>>> query vertices by primary-values: {}",
-                           vertexList);
+        LOG.info(">>>> query vertices by primary-values: {}",vertexList);
```
Comment (javeme) [inline hugegraph-example/src/main/java/com/baidu/hugegraph/example/Example1.java:285]: expect a space after ","

**Review by javeme — APPROVED** [review]

**Review by seagle-yuan — APPROVED** [review]

**Review by seagle-yuan — APPROVED** [review]

**Review by seagle-yuan — APPROVED** [review]

**Review by imbajin — APPROVED** [review]

**Pull request merged**